website/search/elastic_search.py

```diff
@@ -418,11 +416,10 @@ def serialize_contributors(node):
     return {
         'contributors': [
             {
-                'fullname': user.fullname,
-                'url': user.profile_url if user.is_active else None
-            } for user in node.visible_contributors
-            if user is not None
-            and user.is_active
+                'fullname': x['user__fullname'],
+                'url': '/{}/'.format(x['user__guids___id'])
+            } for x in
+            node.contributor_set.filter(visible=True, user__is_active=True).order_by('contributor___order').values('user__fullname', 'user__guids___id')
```

Comment:
In this case, I think `.include` will be faster. The savings in number of queries would outweigh the cost of skipping non-visible and inactive users in Python.

I'll do some comparisons to verify this, though.

I do agree that we need to fix that bug in `include` though. Could lead to some very hard-to-find bugs.